gremlin-go/cucumber/tinkerPopWorld.go

```diff
@@ -115,6 +115,10 @@ func (t *TinkerPopWorld) loadAllDataGraph() {
 }
 
 func (t *TinkerPopWorld) loadEmptyDataGraph() {
+	if _, ok := t.graphDataMap["empty"]; ok {
```

Comment:
The reconnect for empty graphs is actually achieve on `line 716` in `featureSteps_test.go`, where it calls this before every scenario. If you want to share the connection, there is no need to add this check here, just delete that line instead.

Reply:
done in https://github.com/lyndonbauto/tinkerpop/pull/52/commits/e4a7ea2c909eb272e8eeab552a4c559f34471917